packages/packages/commonwell-cert-runner/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metriport/commonwell-cert-runner",
-  "version": "1.4.2",
+  "version": "1.4.3-alpha.3",
```

Comment:
should we bump all these versions to a proper release ver?

Reply:
Yes, I haven't finished updating the description with the release plan yet 😉 

I'll add you as a reviewer once its good to review. 👍